rules/mapping.go

```diff
@@ -183,22 +199,16 @@ func (mc *mappingRule) addSnapshot(
 	rawFilters map[string]string,
 	policies []policy.Policy,
 	cutoverTime int64,
-	opts filters.TagsFilterOptions,
 ) error {
-	filter, err := filters.NewTagsFilter(rawFilters, filters.Conjunction, opts)
-	if err != nil {
-		return err
-	}
-	snapshot := &mappingRuleSnapshot{
-		name:         name,
-		tombstoned:   false,
-		cutoverNanos: cutoverTime,
-		filter:       filter,
-		policies:     policies,
-		rawFilters:   rawFilters,
-	}
+	snapshot := newMappingRuleSnapshotFromFields(
```

Comment:
I took out the filter part because its never used in this case. None of the code in any of the mutation paths actually needs it.